atomic

```diff
@@ -683,6 +682,12 @@ def create_parser(help_text):
     verifyp.add_argument("-v", "--verbose", default=False,
                           action="store_true",
                           help=_("Report status of each layer"))
+    verifyp.add_argument("-V", "--validate", default=True,
+                                action="store_false",
+                                help=_("disable validating system images"))
+    verifyp.add_argument("-g", "--generate", default=False,
```

Comment:
The immutable bit, could be a problem where a user could not remove a file that a hacker had placed on a file system.